src/main/java/com/pusher/client/channel/PrivateEncryptedChannelEventListener.java

```diff
@@ -0,0 +1,10 @@
+package com.pusher.client.channel;
+
+/**
+ * Interface to listen to private encrypted channel events.
+ * Note: This needs to extend the PrivateChannelEventListener because of the ChannelManager clearDownSubscription
+ */
```

Comment:
this comment should be updated to say `handleAuthenticationFailure` as we refactored that method.

Reply:
Have updated.